### DISCUSSION TITLE
How to use PrimeOne for Figma with PrimeNg Ultima?

### DISCUSSION THREAD

**Discussion by mickeyjohn** [primefaces/community#1587] [Figma UI Kit]
We bought PrimeNg Ultima and using it in our web application. 
As mentioned here: https://www.primefaces.org/layouts/ultima-ng, PrimeOne for Figma seems be used to design UI using the compoments of PrimeOne. How to apply the custom theme such as theme-green of Ultima to the PrimeOne Figma UI kit?

Thanks.

**Comment by onursenture** [comment]
Hi, the Ultima template uses our Material Design theme as a component theme, and there is no Material theme in the Figma UI Kit. We recommend using Google's original resources for Material components. https://m2.material.io/design
